test_list_tasks_alphabetic.py

```diff
@@ -0,0 +1,177 @@
+from todo import TodoList
+from pytest import fixture
+
+@fixture
+def empty_todolist():
+    return TodoList()
+
+@fixture
+def grocery_task():
+    return ("Buy grocery", "Medium", "2024-12-02", [], "Holman", False)
+
+@fixture
+def gym_task():
+    return ("Go to gym", "Medium", "2024-12-02", [], "Holman", False)
+
+@fixture
+def gym_task1():
+    return ("go to gym", "Medium", "2024-12-02", [], "Holman", False)
+
+@fixture
+def gym_task2():
+    return ("Ñ Go to gym", "Medium", "2024-12-02", [], "Holman", False)
+
+@fixture
+def num_task1():
+    return ("1 go to gym", "Medium", "2024-12-02", [], "Holman", False)
+
+@fixture
+def num_task2():
+    return ("2 go to gym", "Medium", "2024-12-02", [], "Holman", False)
+
+@fixture
+def one_task_todolist(grocery_task):
+    todo_list = TodoList()
+    todo_list.tasks = [grocery_task]
+    return todo_list
+
+@fixture
+def two_task_todolist(grocery_task, gym_task):
+    todo_list = TodoList()
+    todo_list.tasks = [grocery_task, gym_task]
+    return todo_list
+
+@fixture
+def two_task_todolist_num(num_task1, num_task2):
+    todo_list = TodoList()
+    todo_list.tasks = [num_task1, num_task2]
+    return todo_list
+
+@fixture
+def two_task_todolist_ooo(gym_task, grocery_task):
+    todo_list = TodoList()
+    todo_list.tasks = [gym_task, grocery_task]
+    return todo_list
+
+@fixture
+def two_task_todolist_non_english(gym_task2, grocery_task):
+    todo_list = TodoList()
+    todo_list.tasks = [gym_task, grocery_task]
```

Comment:
This should be todo_list.tasks = [gym_task, grocery_task]. You're passing an invalid reference to test_list_tasks_alphabetic_non_english which is causing that test to fail.

Reply:
Gym task 2 contains the Ñ character which caused the test to fail via an unable to cast to lower case while Gym task did not contain that character and was used in oter spots